.vscode/cspell.json

```
@@ -14,6 +14,8 @@
     "hugo",
     "initializers",
     "Istio",
+    "Jaeger",
+    "jaegertracing",
     "Javadoc",
     "Knative",
     "Kubecon",
```

Similar comment to the one above:

```suggestion
    "kubecon",
```

Then add "KubeCon" to the textlint word set.